cmd/juju/machine/upgradeseries.go

```diff
@@ -281,19 +281,19 @@ func (c *upgradeSeriesCommand) retrieveUnits() ([]string, error) {
 		return nil, errors.Trace(err)
 	}
 
-	var units []string
-	machine, ok := fullStatus.Machines[c.machineNumber]
-	if !ok {
-		return nil, errors.NotFoundf("machine %q", c.machineNumber)
+	instanceID, err := getInstanceID(fullStatus, c.machineNumber)
```

Comment:
I think `machineID` and `getMachineID` might be better given that `instanceID` is suggestive of provider instance ID.